toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1286,6 +1310,40 @@ func runPostInstallScripts(installChroot *safechroot.Chroot, config configuratio
 	return
 }
 
+func setGrubCfgAdditional(grubPath string, kernelCommandline configuration.KernelCommandLine) (err error) {
+	const (
+		extraPattern = "{{.ExtraCommandLine}}"
+		sedDelimiter = "@"
```

Comment:
"@" is a valid character in a kernel command line argument (i.e. crashkernel=256M@0). Recommend to use a different delimiter. Maybe ";" or "^"?

Reply:
We use '@' in a lot of places for the command line, I'll udpate everything to use `` ` ``, I think that should be safe?